distribution/scripts/jmeter/run-backend-tests.sh

```diff
@@ -29,23 +29,32 @@ function initialize() {
 export -f initialize
 
 declare -A test_scenario0=(
-    [name]="backend"
-    [display_name]="Backend"
-    [description]="Directly invoking the backend service"
+    [name]="netty-backend-http"
+    [display_name]="Netty-Backend-HTTP"
+    [description]="This test directly invokes the Netty Backend Service through HTTP protocol"
     [jmx]="backend-test.jmx"
     [protocol]="http"
     [path]="/echo/1.0.0"
     [use_backend]=false
     [skip]=false
 )
 
+declare -A test_scenario1=(
+    [name]="netty-backend-https"
+    [display_name]="Netty-Backend-HTTPS"
+    [description]="This test directly invokes the Netty Backend Service through HTTPS protocol"
+    [jmx]="backend-test.jmx"
+    [protocol]="https"
+    [path]="/echo/1.0.0"
+    [use_backend]=false
```

Comment:
https scenario needs to have following option:  `[backend_flags]="--enable-ssl"`